project/Merging.scala

```diff
@@ -55,6 +55,9 @@ object Merging {
       }
     case "asm-license.txt" | "module-info.class" | "overview.html" | "cobertura.properties" =>
       MergeStrategy.discard
+    // inspired by https://github.com/ergoplatform/explorer-backend/blob/7364ecfdeabeb691f0f25525e577d6c48240c672/build.sbt#L14-L15
```

Comment:
Cool. But what does it do and why?

Reply:
These are annotation classes for [silencing compiler warnings](https://www.scala-lang.org/api/2.13.2/scala/annotation/nowarn.html). Not sure why but a copy of these were being brought in by the fs2 dependencies and they were not identical to the versions in the Scala standard library so the merge was failing. These annotations aren't needed in the uberjar at all so this just throws them out.

Reply:
But since this PR removes fs2VStatsDProxy, I'm confused as to why we are seeing new issues. Is it due to the http4sV upgrade?

Reply:
Yes it's due to the http4s upgrade, either it or its transitive dependencies were bringing in the troublemaker annotation classes.